File: examples/src/Slides/Hyperlinks/AddHyperlink.cs
import aspose.slides as slides
using Aspose.slides.Export
using System
using System.Collections.Generic
using System.Linq
using System.text
using System.Threading.Tasks

namespace CSharp.slides.Hyperlinks
{
    class AddHyperlink
    {
        public static void Run() {

            #ExStart:AddHyperlink

            with slides.Presentation() as presentation:
            {
                shape1 = presentation.slides[0].shapes.add_auto_shape(ShapeType.Rectangle, 100, 100, 600, 50, False)
                shape1.AddTextFrame("Aspose: File Format APIs")
                shape1.text_frame.paragraphs[0].portions[0].portion_format.HyperlinkClick = new Hyperlink("https:#www.aspose.com/")
                shape1.text_frame.paragraphs[0].portions[0].portion_format.HyperlinkClick.Tooltip = "More than 70% Fortune 100 companies trust Aspose APIs"
                shape1.text_frame.paragraphs[0].portions[0].portion_format.font_height = 32

                presentation.save("presentation-out.pptx", slides.export.SaveFormat.PPTX)
            }
            #ExEnd:AddHyperlink
        }
    }
}

File: examples/src/Slides/Hyperlinks/add_hyperlink.py
import aspose.slides as slides


def add_hyperlink(global_opts):
    with slides.Presentation() as presentation:
        shape1 = presentation.slides[0].shapes.add_auto_shape(slides.ShapeType.RECTANGLE, 100, 100, 600, 50, False)
        shape1.add_text_frame("Aspose: File Format APIs")
        shape1.text_frame.paragraphs[0].portions[0].portion_format.hyperlink_click = slides.Hyperlink(
            "https:#www.aspose.com/")
        shape1.text_frame.paragraphs[0].portions[
            0].portion_format.hyperlink_click.tooltip = "More than 70% Fortune 100 companies trust Aspose APIs"
        shape1.text_frame.paragraphs[0].portions[0].portion_format.font_height = 32

        presentation.save(global_opts.out_dir + "hyperlink_add_hyperlink_out.pptx", slides.export.SaveFormat.PPTX)
